src/plugins.cpp

```diff
@@ -62,6 +64,7 @@ namespace Sass {
     typedef Sass_Function_List (*__plugin_load_fns__)(void);
     typedef Sass_Importer_List (*__plugin_load_imps__)(void);
 
+#ifndef _WASM
```

Comment:
Would make sense to disable all plugin related stuff on wasm.
E.g. all plugin related C-API calls and so on. If so I would prefer
to have it behind a separate define, e.g. `ENABLE_PLUGINS`.